scripts/qa_crowbarsetup.sh

```diff
@@ -2572,12 +2579,13 @@ function custom_configuration
             fi
 
             if [ -n "$want_sles12" ] && [ -n "$want_docker" ] ; then
-                proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-compute-docker']" "['${unclustered_sles12plusnodes[0]}']"
+                local computenodes=($(get_non_controller_nodes))
+                proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-compute-docker']" "['$computenodes']"
```

Comment:
I guess that `computenodes` is an array, so not sure that this will work here in there is more that one node. In clase that there is only one, a more implicit `[0]` will help.